.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,10 @@
+contact_links:
+  - about: "Ask and answer user questions about Python on discuss.python.org."
+    name: "Getting help"
+    url: "https://discuss.python.org/c/users/7"
+  - about: "Large feature proposals, for example proposals for new syntax, should be discussed on an ideas forum rather than the issue tracker."
+    name: "Proposing new features"
+    url: "https://discuss.python.org/c/ideas/6"
+  - about: "See here for details on reporting security vulnerabilities in CPython"
+    name: "Reporting security vulnerabilities"
+    url: "https://www.python.org/dev/security/"
```

Comment:
I would reorder all these so that they follow the structure:
```
- name: ...
  about: ...
  url: ...
```

Reply:
Done.